fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/ReindexWorker.java

```diff
@@ -70,18 +77,29 @@ public void start() {
 
     @Override
     public void run() {
+        final var stopwatch = Stopwatch.createStarted();
         while (running) {
             final List<String> ids = manager.getIds();
             if (ids.isEmpty()) {
                 stopThread();
                 break;
             }
+
             int completed = 0;
             int errors = 0;
+            long reportingInterval = REPORTING_INTERVAL_SECS + jitter();
```

Comment:
@pwinckles  Just curious what the expected outcome is here - is the idea that adding random seconds should smooth out the reporting interval - ie make it more regular?

Reply:
Jitter is generally useful to try to reduce resource contention. So, in this case, if multiple works report their status at the same time, one of them will have to wait, briefly. Whereas, if they report at different times, neither have to wait. Here, I don't think it ultimately matters that much with or without it. I can remove it, if you'd prefer.

Reply:
No I think it's fine.  I was just curious.  I appreciate you introducing me to that technique.

Reply:
@pwinckles :  would you mind just adding a comment indicating the purpose for the jitter() in case someone else is unfamiliar that technique?

Reply:
I decided to change to use a reporting thread. Now the status will be reported every 5 minutes and the jitter is irrelevant

Reply:
👍 : I'll try it out now.